catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -50,11 +51,71 @@ class NodeViewTableViewDemoCell: UITableViewCell {
   }
 }
 
+class NodeViewTableViewPrimaryDemoCell: UITableViewCell {
+
+  let containedButton = MDCButton()
+
+  override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    setupContaintButton()
+  }
+
+  required init(coder: NSCoder) {
+    super.init(coder: coder)!
+  }
```

Comment:
This should call setupContainedButton as well

Reply:
Done.